.github/workflows/release_prep_hatch.yml

```diff
@@ -433,7 +447,7 @@ jobs:
           python-version: ${{ env.PYTHON_TARGET_VERSION }}
 
       - name: Run tests
-        run: hatch run test:integration
+        run: hatch run integration-tests:all
```

Comment:
This never gets run because there are no integration tests in this repo, but we should probably fix the command anyway.